ruby/vampires.rb

```diff
@@ -0,0 +1,71 @@
+
+#Employee Total
+puts "How many employees"
+employee_total= gets.chomp.to_i
+#Loop 
+employee=0
+until employee == employee_total
+
+#Name
+puts "what's your name"
+name=gets.chomp.capitalize!
+
+#Age
+puts "how old are you"
+age =gets.chomp.to_i
+
+#Year
+puts "What year were you born"
+year= gets.chomp.to_i
+if 2016 - year == age
+  year= true
+else 
+  year=false
+end
+
+#Garlicbread question
+puts "Do you want garlic bread Y or N"
+garlic_bread = gets.chomp
+if garlic_bread =="y" 
```

Comment:
Calling a comparison operator returns a boolean value of true or false.  If we want to convert the user input into a boolean value we may be inclined to do something like:
```
if user_input == 'y'
	my_var = true
else
	my_var = false
end
```

which is ok, but if we leverage our knowledge of return values we can set our variable my_var to the return of the comparison like:

```my_var = user_input == ‘y’```

much cleaner!